### PR TITLE
docs: Improve readme

### PR DESCRIPTION
**Pull request by Awemeter**
## Description
Improve the readme by removing the A32NX stats and the line about contacting three people.

## Test Results
N/A

## Discord Username
Awemeter#3110

### CODE REVIEW

**Comment by oim1** [comment]
please add changelog

File: README.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
 
-# FlyByWire Discord bot
+# FlyByWire Discord Bot
 
 This is the repository for the FlyByWire Simulations Discord bot.
 
 ## Best Practices
 
 * Please use clear names for your PR's and commands
 * Check [here](https://github.com/flybywiresim/discord-bot/projects/1) to see a list of wanted commands that you may want to work on, or check the #discord-bot channel in the Discord Server
```
Comment (NathanInnes) [inline README.md:17]:
```suggestion
* Check [here](https://github.com/flybywiresim/discord-bot/projects/1) what is being worked on, or check the #discord-bot channel in the Discord Server
* Check the #discord-bot channel for regular discussions about the repository.
```

Just a small edit I suggest

File: README.md [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
 [![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)
 
-# FlyByWire Discord bot
+# FlyByWire Discord Bot
 
 This is the repository for the FlyByWire Simulations Discord bot.
 
 ## Best Practices
 
 * Please use clear names for your PR's and commands
```
Comment (NathanInnes) [inline README.md:16]:
```suggestion
* Please use clear names for your PR's and commands.
```

**Review by oim1 — APPROVED** [review]

**Review by NathanInnes — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
-# FlyByWire Discord bot
+# FlyByWire Discord Bot
 
 This is the repository for the FlyByWire Simulations Discord bot.
 
 ## Best Practices
 
-* Please use clear names for your PR's and commands
-* Check [here](https://github.com/flybywiresim/discord-bot/projects/1) to see a list of wanted commands that you may want to work on, or check the #discord-bot channel in the Discord Server
-* Once you have created your PR, send a message in the #discord-bot channel, or message BenW#8484, █▀█ █▄█ ▀█▀#2123 or oim#0001 on discord, and it will be added to the [projects page](https://github.com/flybywiresim/discord-bot/projects/1). This helps contributors see who is working on what
```
[the remark this answers is not in the document]
Comment (NathanInnes) [reply README.md:22]: Not really.

File: .github/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ## Changelog
 
 Update <small>_ August 2022</small>
```
[the remark this answers is not in the document]
Comment (Awemeter) [reply .github/CHANGELOG.md:3]: Fixed

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (benw202) [reply README.md:22]: > Not really.

Why not? It's fairly useful info to know

[the remark this answers is not in the document]
Comment (NathanInnes) [reply README.md:22]: People do it anyways don't need to specify and is it really a "Best Practice"?

Don't really see the need to specify 🙂

File: .github/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
...
 
+- docs: Improve readme (21/08/2022)
```
Comment (NUTTTTTTTT) [inline .github/CHANGELOG.md:5]:
```suggestion
- docs: improve readme (21/08/2022) 
```

Just to maintain lowercase changelog

**Pull request merged**